app/org/sagebionetworks/bridge/models/surveys/BloodPressureConstraints.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.sagebionetworks.bridge.models.surveys.UIHint.BLOODPRESSURE;
+
+import java.util.EnumSet;
+
+public class BloodPressureConstraints extends Constraints {
+    public BloodPressureConstraints() {
+        setDataType(DataType.STRING);
```

Comment:
Like we talked about offline, looks like DataType is used for mapping a Constraint to a Synapse Column Type, and I didn't see a benefit in adding one for BP